org.freedesktop.Sdk.Extension.ldc.yml

```diff
@@ -0,0 +1,53 @@
+app-id: org.freedesktop.Sdk.Extension.ldc
+branch: '19.08'
+runtime: org.freedesktop.Sdk
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: /usr/lib/sdk/ldc
+modules:
+  - name: ldc-source
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DD_COMPILER="./ldc-bootstrap-bin/bin/ldmd2"
+    sources:
+      - type: archive
+        url: https://github.com/ldc-developers/ldc/releases/download/v1.20.0/ldc-1.20.0-src.tar.gz
+        sha256: 49c9fdfe3a51c978385aae94f2e102f306102f6282215638f2ae3fb9ea8d3ab9
+      - type: archive
+        only-arches:
+          - x86_64
+        url: https://github.com/ldc-developers/ldc/releases/download/v1.20.0/ldc2-1.20.0-linux-x86_64.tar.xz
+        sha256: ab2100228b9396ff1098006f7692d5638f4ebeb07890767499207c8aaf62ff09
+        dest: ldc-bootstrap-bin
+      - type: archive
+        only-arches:
+          - aarch64
+        url: https://github.com/ldc-developers/ldc/releases/download/v1.20.0/ldc2-1.20.0-linux-aarch64.tar.xz
+        sha256: 3b45670b11507fc613661f2fa9bb90611d6a75d8f2b422d2d76788f456a51295
+        dest: ldc-bootstrap-bin
+
+  - name: scripts
+    buildsystem: simple
+    build-commands:
+      - 'cp enable.sh /usr/lib/sdk/ldc'
+    sources:
+      - type: script
+        commands:
+          - 'export PATH="$PATH:/usr/lib/sdk/ldc/bin"'
+          - 'export LIBRARY_PATH="$LIBRARY_PATH:/usr/lib/sdk/ldc/lib"'
+          - 'export LD_LIBRARY_PATH="LD_LIBRARY_PATH:/usr/lib/sdk/ldc/lib"'
+        dest-filename: enable.sh
+
+  - name: appdata
+    buildsystem: simple
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/share/metainfo
+      - cp ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo
+      - appstream-compose --basename=o${FLATPAK_ID} --prefix=${FLATPAK_DEST} --origin=flatpak ${FLATPAK_ID}
+    sources:
+      - type: file
+        path: org.freedesktop.Sdk.Extension.ldc.metainfo.xml
```

Comment:
These both modules are pretty trivial, maybe merge them into one?

Reply:
Ah sure, it just felt like common practice doing it like that looking at other Flatpak manifests.